backend/api/axios-config-decorators.test.js

```diff
@@ -5,26 +5,26 @@ describe('Axios request configuration decorator tests', () => {
   it('should return paging and auth headers', () => {
     const context = {}
     contextProperties.setTokens({ access_token: 'access', refresh_token: 'refresh', authSource: null }, context)
-    contextProperties.setRequestPagination(context, { 'page-limit': 5 })
+    contextProperties.setRequestPagination(context, { offset: undefined, size: 5 })
```

Comment:
we no longer need to pass the headers back to the react app, so use sensible javascript names to set them